src/plugins/ccode/testmod_ccode.cpp

```diff
@@ -0,0 +1,109 @@
+/**
+ * @file
+ *
+ * @brief
+ *
+ * @copyright BSD License (see LICENSE.md or https://www.libelektra.org)
+ */
+
+#include "ccode.hpp"
+
+#include <kdbmodule.h>
+#include <kdbprivate.h>
+
+#include <tests.hpp>
+
+using std::string;
+
+using CppKeySet = kdb::KeySet;
+using CppKey = kdb::Key;
+
+using ckdb::elektraModulesClose;
```

Comment:
It is okay as is but you could also use "using ckdb;".

Reply:
I know, I prefer to only explicitly “import” the functionality I need though.